app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -198,28 +199,32 @@ class ChatActivity : AppCompatActivity(), IChatView {
 
         rv_chat_feed.layoutManager = linearLayoutManager
         rv_chat_feed.setHasFixedSize(false)
-        recyclerAdapter = ChatFeedRecyclerAdapter(this, chatMessageDatabaseList, true)
+        recyclerAdapter = ChatFeedRecyclerAdapter(chatMessageDatabaseList, true, true)
         rv_chat_feed.adapter = recyclerAdapter
 
-        rv_chat_feed.addOnLayoutChangeListener({ _, _, _, _, bottom, _, _, _, oldBottom ->
+        rv_chat_feed.addOnLayoutChangeListener { _, _, _, _, bottom, _, _, _, oldBottom ->
             if (isConfigurationChanged) {
                 isConfigurationChanged = false
             } else {
                 if (bottom < oldBottom) {
                     rv_chat_feed.postDelayed({
-                        var scrollTo = rv_chat_feed.adapter.itemCount - 1
+                        var scrollTo = rv_chat_feed.adapter?.itemCount
+                        if (scrollTo!=null){
+                            scrollTo -= 1
                         scrollTo = if (scrollTo >= 0) scrollTo else 0
                         rv_chat_feed.scrollToPosition(scrollTo)
+                        }
```

Comment:
Correct indentation, use val and add proper spaces around symbols and paranthesis and we are done